test/functional/index.test.js

```diff
@@ -1350,4 +1352,95 @@ describe('Indexes', function() {
       });
     }
   });
+
+  context('should throw an error if commitQuorum specified on MongoDB < 4.4', function() {
```

Comment:
nit: I feel like the `context` here is `commitQuorum` and the tests themselves are like: `should throw an error if commitQuorum specified on db.createIndex`